resources/public/blank.html

```diff
@@ -2,27 +2,29 @@
 <html>
     <head lang="en">
         <meta charset="UTF-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1">
-        <title>OrcPub</title>
-	<link rel="stylesheet" href="/css/style.css" type="text/css">
-	<link rel="stylesheet" href="/css/compiled/styles.css" type="text/css">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<meta name="description" content="The most advanced Dungeons & Dragons 5th Edition (D&D 5e) character builder/generator in the multiverse.">
+        <title>The New OrcPub: D&D 5e Character Builder/Generator</title>
+		<link rel="stylesheet" href="/css/style.css" type="text/css">
+		<link rel="stylesheet" href="/css/compiled/styles.css" type="text/css">
+	<script>
+	  (function(d, s, id){
+	  var js, fjs = d.getElementsByTagName(s)[0];
+	  if (d.getElementById(id)) {return;}
+	  js = d.createElement(s); js.id = id;
+	  js.src = "//connect.facebook.net/en_US/sdk.js";
+	  fjs.parentNode.insertBefore(js, fjs);
+	  }(document, 'script', 'facebook-jssdk'));
+	</script>
     </head>
     <body style="margin:0">
-        <div id="app"/>
-
+        <div id="app">
+			<div style="display:flex;justify-content:space-around">
+				<img src="/image/spiral.gif" style="height:200px;width:200px;margin-top:200px">
+			</div>
+		</div>
         <script src="/js/compiled/orcpub.js"></script>
-	<link rel="stylesheet" href="/font-awesome-4.7.0/css/font-awesome.min.css">
-	<link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
-
-	<script>
-	  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-	  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-	  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-	  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-
-	  ga('create', 'UA-69209720-3', 'auto');
-	  ga('send', 'pageview');
```

Comment:
Please do not remove the Google Analytics tracking.
Wrap it in conditions and allow it to be configured.

Reply:
I dis-agree. These are only used by figwheel.  GA when uberjar is built, it in a different place (and yes THAT needs to be configurable )